replication-api.go

```diff
@@ -109,3 +117,60 @@ func (adm *AdminClient) BucketReplicationDiff(ctx context.Context, bucketName st
 	// Returns the diff channel, for caller to start reading from.
 	return diffCh
 }
+
+// ReplicationMRF represents MRF backlog for a bucket
+type ReplicationMRF struct {
+	NodeName   string `json:"nodeName" msg:"n"`
```

Comment:
Shouldn't we add a Version here, so it is easy to error out when there is an incompatibility in the format and we can make madmin-go supports multiple versions. Maybe we can use major/minor version as well ?

Reply:
done